scripts/installmingw_windows.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+set -eux
+
+MINGW_VERSION="12.2.0.03042023"
+GCC_VERSION="12.2.0"
+
+setup_mingw () {
+    choco upgrade mingw --allow-downgrade --force --version=${MINGW_VERSION}
+}
+
+export PATH="/c/ProgramData/chocolatey/lib/mingw/tools/install/mingw64/bin:$PATH"
+echo "$PATH"
```

Comment:
This is printed here and below, are both necessary?

Reply:
@powersj Just to see what `chocolatey` added to the `$PATH`